src/Tools/ExternalAccess/FSharp/Navigation/IFSharpDocumentNavigationService.cs

```diff
@@ -20,13 +20,15 @@ internal interface IFSharpDocumentNavigationService : IWorkspaceService
         [Obsolete("Call overload that takes a CancellationToken", error: false)]
         bool CanNavigateToLineAndOffset(Workspace workspace, DocumentId documentId, int lineNumber, int offset);
         [Obsolete("Call overload that takes a CancellationToken", error: false)]
+#pragma warning disable RS0060 // API with optional parameter(s) should have the most parameters amongst its public overloads
```

Comment:
💡 These two lines should be reversed if the `#pragma warning` is used for a group  of methods